app/components/top-nav.tsx

```diff
@@ -73,12 +80,23 @@ export default function TopNav() {
                   >
                     About
                   </a>
-                  <a
-                    href="/login"
-                    className="text-gray-300 hover:bg-gray-700 hover:text-white rounded-md px-3 py-2 text-sm font-medium"
-                  >
-                    Login
-                  </a>
+                  {session ? (
+                    <form action="/logout" method="post">
```

Comment:
Nice! This totally works and looks great. Just FYI, Remix also offers a `Form` component that enhances the user experience by avoiding a full page reload. :) We can totally keep it like this for now and re-iterate!

Reply:
Using `Form` looks like an easy win for progressive enhancement and better UX so I'll definitely add it!